torch/quantization/__init__.py

```diff
@@ -14,7 +15,7 @@ def default_eval_fn(model, calib_data):
         model(data)
 
 _all__ = [
-    'QuantWrapper', 'QuantStub', 'DeQuantStub', 'DEFAULT_MODULE_MAPPING',
+    'QuantWrapper', 'QuantStub', 'DeQuantStub',
```

Comment:
this mapping is exposed for users who want to extend the mapping with their own entries. I think it might be useful to expose them?

Reply:
DEFAULT_MODULE_MAPPING is an argument to the convert/prepare_qat/quantize and since we allow for users to change it, not sure if this is needed.

Reply:
So, I am landing it as is in this PR?